src/qqbar/get_acb.c

```diff
@@ -52,41 +60,60 @@ qqbar_get_acb(acb_t res, const qqbar_t x, slong prec)
         arb_init(t);
         fmpz_init(n);
 
+        flint_printf(".1\n");
+
         arb_mul_2exp_si(t, acb_realref(res), wp);
         if (!arb_is_exact(t) && arb_get_unique_fmpz(n, t))
         {
             qqbar_t u;
+
+            flint_printf("detect exact 1\n"); fflush(stdout);
+
             qqbar_init(u);
             qqbar_set_fmpz(u, n);
             qqbar_mul_2exp_si(u, u, wp);
             qqbar_sub(u, x, u);
             if (qqbar_sgn_re(u) == 0)
             {
+                flint_printf("have sgn re = 0\n"); fflush(stdout);
+
                 arb_set_fmpz(acb_realref(res), n);
                 arb_mul_2exp_si(acb_realref(res), acb_realref(res), wp);
             }
             qqbar_clear(u);
         }
 
+        flint_printf(".2\n"); fflush(stdout);
+
         arb_mul_2exp_si(t, acb_imagref(res), wp);
         if (!arb_is_exact(t) && arb_get_unique_fmpz(n, t))
         {
             qqbar_t u;
+
+            flint_printf("detect exact 2\n"); fflush(stdout);
+
             qqbar_init(u);
             qqbar_i(u);
             qqbar_mul_fmpz(u, u, n);
             qqbar_mul_2exp_si(u, u, wp);
             qqbar_sub(u, x, u);
+
             if (qqbar_sgn_im(u) == 0)
             {
+                flint_printf("have sgn im = 0\n"); fflush(stdout);
+
                 arb_set_fmpz(acb_imagref(res), n);
                 arb_mul_2exp_si(acb_imagref(res), acb_imagref(res), wp);
             }
             qqbar_clear(u);
         }
 
+        flint_printf(".3\n"); fflush(stdout);
```

Comment:
It should occur here. I have currently pruned all other prints in a new job: https://github.com/albinahlback/flint2/actions/runs/4469612730/jobs/7851970684

Reply:
```
.2
detect exact 2
wp = 4567
n = -127356229551470686006317059333789205746744145371041802895070746336990527137735077348718885757633710613123221306421375381520819705469364014798907578813124137345860612586525783149585468459223144506531416367864813691549209985965936693523916177162286265758554347936673041856623500349146552099024091933274326583091436445588256826355694481122645904176831044059960210918483375838011164680093380568479707494529744871637053567849334340419066148186720937646478770260627349108102323410166294713496640001145251082159654181504740164387692152778828735405096729320142655551717443773567884571702025581458951383408787946763710681441011870671204803999965031910328620340501708815124209724718861046661826095689620903566975425686515730704742048557160008479752991782892492679687459637576117511454477690989176070124282040793028310052665335683125045126172560964123551601598593215976100481147654497134447078194413956631202321654572923770372567160005050317618818381781549887379057128831129757073124751105437274130709994808141829182780412343245480049879358531710928522096977023984527681974808146391347894184408548165856330092177602558860171693453787406672751822855693573443032200008668262210447857023002171351727295843338049388071730287598421189124829666543028578072980531191948846942203349033144921169488175614429303007228584055662352066643657736288289800901303718154524221771313188099791211784107458560
x = deg 4 [1065600, 16800, 2164, 84, 1] [4.31797780234432546177728574612429583432553387574778628387731323738049928794826985590608840288396741099845213374800875383711112827713565235617908867480131014996840752173779801932940170205985104451140649044799222672141608507766298972550929410133375549055453135412526892953574153585875698462352117649946028381426840927121143304231270202923696537723478971519611635712249577221504908061044723839787163600749321418130057210916657210671704596573048509425369990566700489232601220798681894818654338796280122427785033004545766131029501275005212614000913260599359072548826827143611487519785345738540558533704058296206972548796280321675532290176919098349654243404569391719218686049897882447378465480 +/- 3.48e-687] + [-20.0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000 +/- 1.80e-686]*I
pre qqbar_i
pre qqbar_mul_fmpz
pre qqbar_mul_2exp_si
pre qqbar_sub
##[error]Process completed with exit code 1.
```

Reply:
So it is inside `qqbar_sub`. I get the same values on my system, but it does not error of course.

Reply:
Looks like `fmpz_poly_factor` fails within `qqbar_binary_op_without_guess`

Reply:
It might be due to lack of memory. In `qqbar_binary_op_without_guess`, `H` contains a coefficient which should take up about 73 kB in memory.

Reply:
The maximum amount of memory usage I get for the whole test is 13 MB, so that shouldn't be a problem.